_posts/en/newsletters/2022-07-06-newsletter.md

```diff
@@ -0,0 +1,194 @@
+    before adding the complexity of payments for onion messages.
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+FIXME:harding to update Tuesday
+
+- LDK
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo], [Core
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #24836][] adds a regtest-only RPC, `submitpackage` to
+  help L2 protocol and application developers who intend to use
+  package relay in the future test their transactions against the
+  Bitcoin Core default package policy.  The current policy is outlined
+  [here][packages doc]. This RPC can also be used to test future
+  additions and changes, such as the proposed package RBF rules.
+
+- [Bitcoin Core #22558][] adds support for [BIP371][]’s additional [PSBT][topic
+  psbt] fields (see [Newsletter #155][news155 psbt extensions]) for [taproot][topic taproot].
+
+- [Core Lightning #5281][] adds support for specifying the `log-file`
+  configuration option multiple times to write to multiple log files.
+
+- [LDK #1555][] updates its pathfinding code to slightly prefer routing
+  through channels which advertise that they won't accept payments
+  larger than half the amount of money committed to the channel.  This
+  is believed to provide a slight privacy improvement by limiting the
+  amount of balance information a third party can discover by probing a
+  channel (sending a payment ([HTLC][topic htlc]) they don't intend to
+  settle).  If a set of payments up to the total amount of a channel can
+  be sent, then a prober can learn nearly the exact balance of the
+  channel by just trying different sets of payments until all parts are
+  accepted.  But, if the set of payments which can be sent is limited to
+  half the channel balance, it's harder for the prober to determine
+  whether payments are being rejected because of a lack of funds on one
+  side of the channel or because of the advertised limit (the
+  `max_htlc_in_flight` limit).
+
+- [LDK #1550][] provides the ability for users to add a list of nodes to
+  a local banlist which will prevent pathfinding from routing payments
+  through those nodes.
+
+- [LND #6592][] adds a new `requiredreserve` RPC to the wallet subserver
+  that prints the number of satoshis the wallet is reserving in UTXOs it
+  unilaterally controls to fee bump [anchor outputs][topic anchor
+  outputs] if necessary.  An additional `--additionalChannels` parameter
```

Comment:
Suggesting for clarity:

An additional `--additionalChannels` RPC parameter, which takes a integer argument...